File: src/main/java/org/example/bookfd/repos/UserRepo.java
package org.example.bookfd.repos;

import org.example.bookfd.domain.User;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface UserRepo extends JpaRepository<User,Long> {
   User findById(int id);
   List<User> findByIdIn(List<Integer> listUsersId);

}

File: src/main/java/org/example/bookfd/repos/UserRepo.kt
package org.example.bookfd.repos

import org.example.bookfd.domain.User
import org.springframework.data.jpa.repository.JpaRepository

interface UserRepo : JpaRepository<User?, Int?> {
    fun findById(id: Int): User?
    fun findByIdIn(listUsersId: List<Int?>?): List<User?>?
}
